src/cmd/cmd.cpp

```diff
@@ -311,7 +311,7 @@ int main(int argc, char **argv)
     qputenv("OPENSSL_CONF", opensslConf.toLocal8Bit());
 #endif
 
-    qsrand(QTime::currentTime().msec() * QCoreApplication::applicationPid());
+    qsrand(QDateTime::currentMSecsSinceEpoch());
```

Comment:
Oh, makes sense to leave in the pid then. My complaint was that msec() just has values between 0-999.